docs/packages/Package Development/package development main.rst

```diff
@@ -3,20 +3,18 @@
 Package Development: A "How-to Guide" for Developing Packages for Others to Use
 ================================================================================
 
-The purpose of this page is to get your mesa package sharing as quickly as possible. The authoritative guide for python package development is through the `Python 
-Packaging User Guide <https://packaging.python.org/>`_. This will take you through the entire process necessary for getting you package on the Python Package Index.
+The purpose of this page is to get your mesa package sharing as quickly as possible. 
 
 This "How-to Guide" uses GitHub to walk you through the process. However, other repositories will be able to provide similar services.
 
-
-Package Development Checklist (basic): Sharing your modules in seven steps
+Package Development Checklist (basic): Sharing your package in seven steps
```

Comment:
If this is "basic" ... where is advanced? If there is no advanced yet, then we should delete "basic".